bananas/admin/extension.py

```diff
@@ -130,8 +130,8 @@ def get_context(self, request, **extra):
         context.update(
             {
                 "app_label": opts.app_label,
-                "model_name": force_text(opts.verbose_name_plural),
-                "title": force_text(opts.verbose_name_plural),
+                "model_name": force_str(opts.verbose_name_plural),
+                "title": force_str(opts.verbose_name_plural),
```

Comment:
Is `force_str` still needed here? Seems like `force_text` has been used for earlier Python 2 compatibility and that we can now safely assume that the attributes will be `str`. It would only make sense to use `force_str` here if Django explicitly allowed usage of `bytes` for these attributes, IMO.

Reply:
Can't really remember either why these are forced, but it could be to enforce lazy gettext attributes to be real strings in the context.

Reply:
Ah, that makes sense! 👍